src/package_manager/apt.ts

```diff
@@ -105,7 +107,8 @@ async function determineDistribCodename(): Promise<string> {
  * @returns Promise<number> exit code
  */
 export async function installAptDependencies(): Promise<number> {
-	let aptPackages: string[] = aptDependencies;
+	let aptPackages: string[] = installConnext ?
```

Comment:
To our other conversation - I would definitely prefer to avoid globals. I just meant that this generic package manager utility function shouldn't have to know anything about "connext" specifically, this would be better enabled as allowing for an arbitrary prefix string.

Reply:
I still don't really understand what you are asking for...I'm not that familiar with typescript tbh.

If you'd like it not to be connext specific then probably doing it the way I had it before with a string array for any non-commercial package people would want to add in the future makes sense, no?

Reply:
Let's change it back to 

```
function installAptDependencies(installConnext: bool = false): Promise<number> {
```

The shared mutable global state across modules is definitely not what we want :)

Then, `setup-ros-linux` can call `await apt.installAptDependencies(installConnext)`  without having to mutate global state.

Sorry for any confusion

Reply:
no need to apologize! this is mostly me just not knowing ts i think. so sorry from my side  😄